### PR TITLE
Synchronize

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -146,55 +146,60 @@ class _FadeAnimationState extends State<FadeAnimation>
...
+  _PlayerLifeCycleState createState() => new _PlayerLifeCycleState();
 }
 
 class _PlayerLifeCycleState extends State<PlayerLifeCycle> {
-  final VideoWidgetBuilder videoWidgetBuilder;
-  Future<VideoPlayerController> video;
+  VideoPlayerController controller;
 
-  _PlayerLifeCycleState(this.videoWidgetBuilder);
+  _PlayerLifeCycleState();
```
Comment (mravn-google) [inline example/lib/main.dart]: Remove?

File: lib/video_player.dart [hunk 2]
```diff
@@ -24,127 +24,152 @@ class VideoPlayerValue {
...
+    await _applyPlayPause();
+  }
+
+  Future<Null> _applyLooping() async {
+    if (!value.initialized || isDisposed) {
       return;
     }
-    await _channel.invokeMethod(
-        'setLooping', {'textureId': _textureId, 'looping': value});
+    _channel.invokeMethod('setLooping',
```
Comment (mravn-google) [inline lib/video_player.dart]: Confusing that we do not `await` here.

File: example/lib/main.dart [hunk 3]
```diff
@@ -146,55 +146,60 @@ class _FadeAnimationState extends State<FadeAnimation>
...
+        print(controller.value.errorDescription);
       }
     });
+    controller.initialize();
+    controller.setLooping(true);
+    controller.play();
+  }
+
+  @override
+  void deactivate() {
```
Comment (mravn-google) [inline example/lib/main.dart]: Remove?

File: example/lib/main.dart (hunk 1 above)
Comment (sigurdm) [reply example/lib/main.dart]: Done

File: example/lib/main.dart (hunk 3 above)
Comment (sigurdm) [reply example/lib/main.dart]: Done

**Pull request merged**